APMrover2/GCS_Mavlink.cpp

```diff
@@ -1082,6 +1082,15 @@ void GCS_MAVLINK_Rover::handleMessage(mavlink_message_t* msg)
             break;
         }
 
+        case MAV_CMD_ACCELCAL_VEHICLE_POS:
+            if (rover.ins.get_acal()->gcs_vehicle_position(packet.param1)) {
```

Comment:
I'd prefer the result be set to failed then check to pass instead of an else. Matches many other examples of that use as well.